src/components/intro-page/hooks/use-intro-page-content.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { getFrontPageContent } from '../requests'
+import { useFrontendBaseUrl } from '../../../hooks/common/use-frontend-base-url'
+
+const MARKDOWN_WHILE_LOADING = ':zzz: {message}'
+const MARKDOWN_IF_ERROR = ':::danger\n' +
+  '{message}\n' +
+  ':::'
+
+export const useIntroPageContent = (): string => {
+  const { t } = useTranslation()
+  const [content, setContent] = useState<string>(() => MARKDOWN_WHILE_LOADING.replace('{message}', t('landing.intro.markdownWhileLoading')))
```

Comment:
Wouldn't it be more like the rest of the application if we, would define MARKDOWN_WHILE_LOADING and MARKDOWN_IF_ERROR inside the hook, but directly put the correct translation inside, instead of this unique replaceing?

Reply:
1. The messages are not inside the hook, because they're static.
2. imho this approach is much more readable.